extensions/sql-database-projects/package.json

```diff
@@ -464,8 +464,11 @@
     "vscode-languageclient": "^5.3.0-next.1",
     "vscode-nls": "^4.1.2",
     "which": "^2.0.2",
-    "xml-formatter": "^2.1.0",
-    "xmldom": "^0.6.0"
+    "xml-formatter": "2.1.0",
+    "xmldom": "^0.6.0",
+    "@azure/arm-resources": "^5.0.0",
```

Comment:
It would be great not to have the dependency on these packages. What you can do is instead of having the @azure types in the typings file is just to create our own types defined in the d.ts that has the same properties. Kind of annoying but much better than people needing to use these packages when they don't need to (and could result in larger VSIX packages)

Reply:
(and type checking should ensure that what we get back from those packages maps to the same properties)